### DISCUSSION TITLE
mkdocs

### DISCUSSION THREAD

**Discussion by m-GDEV** [selfhst/icons#111] [New Icon Request]
https://www.mkdocs.org/

https://github.com/mkdocs/mkdocs

Not sure if they have an official logo but their Github profile has a logo.

**Comment by shollyethan** [comment]
I've reached out to the MkDocs teams to see if they have an icon they'd like me to use. Will respond here when I hear back!

**Comment by shollyethan** [comment]
MkDocs has been added -- thanks!
